### PR TITLE
RDBC-658 AddDatabaseNodeOperation: Don't add node param to request wh…

### PR DESCRIPTION
**Pull request by Danielle9897**
…en node is undefined

### CODE REVIEW

File: src/ServerWide/Operations/AddDatabaseNodeOperation.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ class AddDatabaseNodeCommand extends RavenCommand<DatabasePutResult> implements
     createRequest(node: ServerNode): HttpRequestParameters {
         let uri = node.url + "/admin/databases/node?name=" + this._databaseName;
 
-        if (node) {
+        if (node && this._node) {
```
Comment (ml054) [inline src/ServerWide/Operations/AddDatabaseNodeOperation.ts]: why we need `if (node` check ?

Comment (ml054) [reply src/ServerWide/Operations/AddDatabaseNodeOperation.ts]: If node is null/undefined it will throw any way in line 46

Comment (Danielle9897) [reply src/ServerWide/Operations/AddDatabaseNodeOperation.ts]: * fixed
* refactored` this._node` to `this._nodeTag` (I hope that is ok)

**Review by ml054 — APPROVED** [review]

**Comment by Danielle9897** [comment]
Using the env-var worked great
Tests pass for me locally

**Comment by ml054** [comment]
I knew it will help. I caught myself so many times on that. :D

File: test/Issues/RDBC_658.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    let testContext: ClusterTestContext;
+
+    beforeEach(async () => testContext = new ClusterTestContext());
+    afterEach(async () => testContext.dispose());
+
+    describe("Add database node", function () {
+
+        it("Add random database node", async () => {
+            try {
+                testContext = new ClusterTestContext();
```
Comment (ml054) [inline test/Issues/RDBC_658.ts]: please compare this test and any other with cluster test context.

https://github.com/ravendb/ravendb-nodejs-client/blob/7db833e14a10c92cab91ff8b9276deb7744e5f17/test/Ported/Issues/RDBC_538.ts#L6

IE. we don't override textContext variable in test

I bet that's the reason tests fails

**Pull request merged**